server/src/graphql/api.ts

```diff
@@ -57,7 +57,7 @@ export const graphqlRoot: Resolvers<Context> = {
       // This can be made more flexible
       const result = await getManager()
         .createQueryBuilder(Art, "art")
-        .where("(abs(art.location.lat - :lat) < 20) AND (abs(art.location.lng - :lng) < 20)", { lat: loc.lat, lng: loc.lng })
+        .where("(abs(art.location.lat - :lat) < 0.02) AND (abs(art.location.lng - :lng) < 0.02)", { lat: loc.lat, lng: loc.lng })
```

Comment:
Is this a large enough radius?

Reply:
Yes, the unit is degrees [latitude and longitude](https://en.wikipedia.org/wiki/Decimal_degrees#Precision).
I will admit that the behavior may differ with distance from the equator

Reply:
lgtm 👌👌